scheds/rust/scx_bpfland/src/bpf/main.bpf.c

```diff
@@ -333,6 +356,18 @@ static int dispatch_direct_cpu(struct task_struct *p, s32 cpu, u64 enq_flags)
 
 	scx_bpf_dispatch_vtime(p, dsq_id, slice, vtime, enq_flags);
 
+	/*
+	 * If the CPU has gone offline notify that the task needs to be
+	 * consumed from another CPU.
+	 */
+	offline = offline_cpumask;
+	if (!offline)
+		return 0;
+	if (bpf_cpumask_test_cpu(cpu, cast_mask(offline))) {
```

Comment:
Hmm... this is a bit tricky. I think this can be a lot simpler if we had a way to wait a RCU grace period.

Reply:
It is quite ugly, also offline_cpumask is allocated once at init and it goes away only when the scheduler exits (it's never re-allocated), this is just to make the verifier happy...